packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -2,6 +2,32 @@
 
 The terra-functional-testing library is a utility for developing automation tests. The library extends the [WebdriverIO Framework](https://webdriver.io/) to facilitate automating accessibility and functional testing for Terra projects.
 
+## Viewports
+
+Tests can be executed in any of the following supported viewports:
+- 'tiny' - width: 470; height: 768
+- 'small' - width: 622; height: 768
+- 'medium' - width: 838; height: 768
+- 'large' - width: 1000; height: 768
+- 'huge' - width: 1300; height: 768
+- 'enormous' - width: 1500; height: 768
+
+### Terra.describeViewports
```

Comment:
Should this live in the TerraService docs?

Reply:
I put this here because this explains how to write a wdio test using describeViewport.

Reply:
We'll probably need documentation in both places, but I think it makes sense to document how to use the commands in the WDIO Testing section